metron-platform/metron-writer/src/main/java/org/apache/metron/writer/hdfs/HdfsWriter.java

```diff
@@ -111,6 +121,16 @@ public BulkWriterResponse write(String sourceType
     return response;
   }
 
+  public SyncPolicy createSyncPolicy(int batchSize) {
```

Comment:
This is extraneous isn't it? It's handled in init() and glancing through, I don't see it getting called elsewhere.